### ISSUE TITLE
Representation Chapter: challenging explanation of ALL uncountability 

### ISSUE THREAD

**Issue by marwahaha** [boazbk/tcs#562]
I was able to follow the one-to-one map from {0,1}^\inf to ALL, but I feel like it could be more plainly stated.

The definition of a one-to-one map is that every element in the domain can be mapped to a unique element in the codomain.

In plain English, I can map {0,1}^\inf to ALL by taking a function f: N -> {0,1} (i.e. 0 -> 0, 1 -> 0, 2 -> 1, 3 -> 0, 4 -> 1...) and identifying it with a function g: {0,1}* -> {0,1} that extends f (i.e. 0 -> 0, 1 -> 0, 10 -> 1, 11 -> 0,  100 -> 1...) and sending all other bitstrings to 0. This is one-to-one because g's output depends on f; if f is a different function, than g will be different too.

I recognize this is what you do in the proof. However, I feel like this is worth explaining in words. With much jargon in the proof (ALLtS, SEQtALL, there exists syntax, f vs F), I am easily distracted from the main point.

**Comment by boazbk** [comment]
Thanks! 
Just tried to fix this - please  take a look.
Let me know if there are other proofs where the syntax distracts from the main idea

**Comment by marwahaha** [comment]
Looks great, thanks.
